Sources/Camera/ZLCustomCamera.swift

```diff
@@ -1032,7 +1077,13 @@ open class ZLCustomCamera: UIViewController {
         }
         do {
             try device.lockForConfiguration()
-            device.videoZoomFactor = zoomFactor
+            if cameraConfig.enableWideCameras {
+                let minZoomFactor = device.minAvailableVideoZoomFactor
```

Comment:
```swift
@available(iOS 11.0, *)
open var minAvailableVideoZoomFactor: CGFloat { get }
```

The minAvailableVideoZoomFactor property supports iOS 11 at the minimum, and an error will be reported when running here.

I noticed that when you used enableWideCameras, there were several places where @available(iOS 13.0, *) was judged, so I think it would be better to mark the enableWideCameras property as @available(iOS 13.0, *), what do you think?

Reply:
Hey @longitachi, yes, it makes sense, I wanted this PR to have as few LOC changes as possible since an explicit availability check will still be needed for `AVCaptureDevice.DeviceType`. I provided changes, thanks!